addons/binding/org.openhab.binding.netatmo/ESH-INF/thing/channels.xml

```diff
@@ -249,19 +248,19 @@
 	</channel-type>
 
 	<channel-type id="heatIndex">
-		<item-type>Number</item-type>
+		<item-type>Number:Temperature</item-type>
 		<label>Heat Index</label>
 		<description>Computed Heat Index</description>
 		<category>Temperature</category>
-		<state readOnly="true" pattern="%.0f" />
+		<state readOnly="true" pattern="%.0f %unit%" />
 	</channel-type>
 
 	<channel-type id="dewPoint" advanced="true">
-		<item-type>Number</item-type>
+		<item-type>Number:Temperature</item-type>
 		<label>Dewpoint</label>
 		<description>Computed Dewpoint Temperature</description>
 		<category>Temperature</category>
-		<state readOnly="true" pattern="%.1f °C" />
+		<state readOnly="true" pattern="%.1f %unit%" />
 	</channel-type>
 
 	<channel-type id="dewPointDepression" advanced="true">
```

Comment:
What about this channel? Doesn't it contain a temperature, too?

Reply:
Exactly. It is a delta between two temperature. Hence it is a temperature, too (see https://en.m.wikipedia.org/wiki/Dew_point_depression)